src/main/controller/UserController.ts

```diff
@@ -181,7 +181,7 @@ export class UserController {
               where: {
                 id: In(promotionIds),
               },
-              relations: ['discount', 'schedules'],
+              relations: ['discount', 'schedules', 'restaurant'],
```

Comment:
Hmm, I was thinking I might as well return everything, including the restaurant (which now only entails lat/lon). No specific reason but I was thinking of keeping it consistent with GET/promotions and GET/promotions/:id

Reply:
I think it might be nice if we end up adding a section on the homepage where you can see your saved promotions with the map? Feature doesn't exist yet but could work (?)

Reply:
Ahhh okay makes sense! But yeah @ld357 that's a cool idea, I remember that idea was brought up so maybe we will end up needing this in the future 👀